runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -2427,6 +2427,19 @@ J9::SymbolReferenceTable::findOrCreateObjectEqualityComparisonSymbolRef()
    return symRef;
    }
 
+TR::SymbolReference *
+J9::SymbolReferenceTable::findOrCreateNonNullableArrayNullStoreCheckSymbolRef()
+   {
+   TR::SymbolReference *symRef = element(nonNullableArrayNullStoreCheckSymbol);
+   if (symRef != NULL)
+      return symRef;
+
+   symRef = self()->findOrCreateCodeGenInlinedHelper(nonNullableArrayNullStoreCheckSymbol);
+   symRef->setCanGCandReturn();
```

Comment:
Can this helper call actually GC and return ? I was thinking a GC may be possible only if an exception were possible, i.e. otherwise it is just a test of the value being null ?

Reply:
Thanks for catching that.  Yes, I'll remove the call to `setCanGCandReturn()` and leave just the call to `setCanGCandExcept()`.

Reply:
Fixed in commit 9109a173c2c22cf1250740e000ed5bf9dbadf69d.  I can squash this commit when all reviews are completed.